addons/mail/static/src/new/views/fields/list_activity/list_activity.js

```diff
@@ -16,7 +16,7 @@ export class ListActivity extends Component {
             return this.props.record.data.activity_summary;
         }
         if (this.props.record.data.activity_type_id) {
-            return this.props.record.data.activity_type_id[1 /** display_name **/];
+            return this.props.record.data.activity_type_id[1 /** display_name */];
```

Comment:
this is not a js doc either, could be single *

Reply:
I'll merge, you can do change in a separate PR.

Reply:
https://github.com/odoo-dev/odoo/pull/1541